rpm_spec/manageiq.spec.in

```diff
@@ -5,17 +5,21 @@
 
 %global app_root /var/www/miq/vmdb
 %global org_root /opt/%{org_name}
+%global ansible_venv_root /var/lib/manageiq
 %global appliance_root %{org_root}/%{name}-appliance
 %global gemset_root %{org_root}/%{name}-gemset
 %global manifest_root %{org_root}/manifest
 
+%global ansible_venv_builddir %{name}-ansible-venv-%{version}
 %global appliance_builddir %{name}-appliance-%{version}
 %global core_builddir %{name}-core-%{version}
 %global gemset_builddir %{name}-gemset-%{version}
 %global manifest_builddir %{name}-manifest-%{version}
 
 %global debug_package %{nil}
 %global __requires_exclude ^\/usr\/bin\/jruby
+# Turn off the brp-python-bytecompile automagic
+%global _python_bytecompile_extra 0
```

Comment:
I wasn't sure if this should live in the ansible-venv package or not - feels like it's something we globally want to avoid, so I put it here.